README.md

```diff
@@ -193,6 +193,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 
 ## Database Drivers/Clients
  * [arangocr](https://github.com/solisoft/arangocr) - ArangoDB client
+ * [couchdb.cr](https://github.com/TechMagister/couchdb.cr) - CouchDB client
```

Comment:
@Schniz @TechMagister looks like it is not maintained. This shard doesn't even compile